$OpenBSD: patch-mobile_licenses_CMakeLists_txt,v 1.2 2013/12/08 19:13:33 zhuk Exp $
TODO: Create/reuse some sort of FindDBLATEX.cmake and use it
instead of find_program() call.
--- mobile/licenses/CMakeLists.txt.orig	Wed Aug 28 21:05:44 2013
+++ mobile/licenses/CMakeLists.txt	Thu Nov  7 16:50:47 2013
@@ -1,6 +1,7 @@
 find_program(DBLATEX_EXECUTABLE dblatex)
 add_feature_info("Kontact Touch license information" DBLATEX_EXECUTABLE "Dblatex is needed for building the Kontact Touch license information.")
 
+set(DBLATEX_EXECUTABLE FALSE)
 if(DBLATEX_EXECUTABLE)
   add_custom_command(
     OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/licenses.pdf
